src/locales/ar.json

```diff
@@ -73,5 +73,11 @@
         }
       ]
     }
+  },
+  "contact": {
+    "contactUsHeaderSection": {
```

Comment:
I'd prefer if you can get this translation before you submit. normally we don't submit code that's half complete because there's a very high possibility someone will forget to do the other half. so let's get this in then I'll approve, rest of code is fine though

Reply:
Thanks for reviewing! I have added Arabic translation. PTAL